vaadin-grid-flow-parent/vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -136,10 +136,6 @@ import { isFocusable } from '@vaadin/grid/src/vaadin-grid-active-item-mixin.js';
           itemkey: grid.getItemId(targetCache.items[scaledIndex]),
           level: targetCache.getLevel()
         });
-        // sort by ascending scaledIndex and level
-        ensureSubCacheQueue.sort(function(a, b) {
-          return a.scaledIndex - b.scaledIndex || a.level - b.level;
-        });
```

Comment:
This sorting logic didn't really add up: `scaledIndex` is an item's index under its own level/cache, so using it for sorting universally doesn't really make sense. Removing this invalid sorting logic is essential for making the preloading feature work.